lib/home/widgets/task_list.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+import 'package:tasks_repository/tasks_repository.dart';
+
+
+class TaskCard extends StatelessWidget {
```

Comment:
this file contains TaskCard class so its name should be `task_card.dart` instead of `task_list.dart`

Reply:
you did not change it

Reply:
I change it, commit [Rename file name to task_card.dart](https://github.com/kamilgadek/to_do_list_kamilg/pull/21/commits/73ae07957c411bbb158d8011e421a48abeda8b97)